Localization/StringsConvertor/Sources/StringsConvertor/Parser.swift

```diff
@@ -43,7 +43,12 @@ extension Parser {
                             .map {
                                 switch keyStyle {
                                 case .infoPlist:        return $0
-                                case .swiftgen:         return $0.capitalized
+                                case .swiftgen:
+                                    if $0 == "a11y" {
+                                        return "A11y"
+                                    } else {
+                                        return $0.capitalized
+                                    }
```

Comment:
I'm a bit confused. You changed the keys in `app.json`, which means: This isn't necessary anymore, right? Or is there anything I don't see? Sorry for being a bit nitpicky 🙃

Reply:
Yep, reverted!